engine/src/parser/expression_utils.cpp

```diff
@@ -220,6 +222,10 @@ cudf::type_id get_output_type(operator_type op, cudf::type_id input_left_type, c
 	case operator_type::BLZ_STR_SUBSTRING:
 	case operator_type::BLZ_STR_CONCAT:
 		return cudf::type_id::STRING;
+	case operator_type::BLZ_STR_LOWER:
+		return cudf::type_id::STRING;
+	case operator_type::BLZ_STR_UPPER:
+		return cudf::type_id::STRING;
```

Comment:
would it make sense to add these cases under the `return cudf::type_id::STRING;` above?

Reply:
Yep, good point. Just fixed